symtabAPI/src/dwarfWalker.h

```diff
@@ -233,7 +237,9 @@ class DwarfWalker : public DwarfParseActions {
 
     } Error;
 
-    DwarfWalker(Symtab *symtab, Dwarf* dbg);
+    using ParsedFuncs = tbb::concurrent_hash_map<FunctionBase *, bool>;
```

Comment:
I know that the old implementation used a `std::set<FunctionBase*>`, but we have had a great many bugs tied to using pointers as keys in associative containers. Now would be a great time to excise this one. I _think_ the easiest solution is to make a hasher to capture the symbol name, address, and module name to make them unique.